store/tikv/error/errcode.go

```diff
@@ -33,8 +33,6 @@ const (
 
 	CodeTiKVStoreLimit = 9008
 
-	CodeTiKVStaleCommand          = 9010
```

Comment:
seem here we won't use the Error code later? do we need to update the doc?

Reply:
NO，here we just remove this code from `tikv-client`, and the code still exists in `tidb`.